docs/config.toml

```diff
@@ -37,3 +37,20 @@ publishDir = "../out/docs/"
 [[menu.extra]]
     name = "submit an issue"
     url = "https://github.com/workspacer/workspacer/issues/new"
+
+[[menu.footer]]
+    name = "Workspacer"
+    url = "https://github.com/workspacer/workspacer"
+    weight = 1
+    
+[[menu.footer]]
+    name = "Rick Button"
+    url = "https://github.com/rickbutton"
+    weight = 2
+    
+    
+[[menu.footer]]
+    name = "Website"
+    url = "https://rickbutton.me"
```

Comment:
Apparently his new website is https://button.dev/, in case we want to keep this reference this should be updated